addon.py

```diff
@@ -56,8 +56,10 @@ def show_dir(subdir=''):
 @plugin.route('/conference/<conf>')
 def show_conference(conf):
     data = None
+    relive = None
     try:
         data = http.fetch_data('conferences/' + conf)
+        relive = http.fetch_relive_index().by_conference(conf)
```

Comment:
Maybe put this into its own try-catch-block, so an error in fetching relive data doesn't prevent the whole conference from showing?